File: 8 kyu/DNA to RNA Conversion/solution.java
public class Bio{
    public String dnaToRna(String dna){
        return dna.replace("T", "U");
    } 
}

_____________________________
import java.io.*;

public class Bio {
    public String dnaToRna(String dna){
        return dna.replaceAll("T", "U");
    } 
}

_____________________________
public class Bio {
    public String dnaToRna(String dna) {

    String rna = "";
       
     // For every letter in String called DNA
     for(int i = 0; i < dna.length(); i++) {
     
     // get the letter at index i
      char letter = dna.charAt(i);
     
      // if letter != T,
      //    add letter to rna
      // else 
       //   add U 
       if(letter != 'T') {
         rna += letter;
       } else {
         rna += 'U';     
     }
     
     }
     

       return rna;  // Do your magic!
    }
}

_____________________________
class Bio {
  static String dnaToRna(String dna) {
    return dna.replace("T", "U");
  }
}

_____________________________
public class Bio {
    public String dnaToRna(String dna) {
        String result = dna;
        return result.replace('T','U');
    } 
}

File: 8 kyu/DNA to RNA Conversion/solution.go
package kata

import (
  "strings"
)

func DNAtoRNA(dna string) string {
  return strings.Replace(dna, "T", "U", -1)
}

_____________________________
package kata

import "strings"

func DNAtoRNA(dna string) string {
  return strings.ReplaceAll(dna, "T", "U")
}

_____________________________
package kata
import ."strings"

func DNAtoRNA(dna string) string {
  return ReplaceAll(dna, "T", "U")
}

_____________________________
package kata

func DNAtoRNA(dna string) string {
  var response string

  for _, letter := range dna{
    if letter == 'T'{
      letter = 'U'
    }
    response+=string(letter)
  }

  return response
}
